sig-network/link-state-proposal.md

```diff
@@ -0,0 +1,211 @@
+# VEP #2: Adding Link State Management for vNICs
+
+## Release Signoff Checklist
+Items marked with (R) are required *prior to targeting to a milestone / release*.
+
+- [ ] (R) Enhancement issue created, which links to VEP dir in [kubevirt/enhancements] (not the initial VEP PR)
```

Comment:
^ As above, I'll try to make this more clear now.

Reply:
I've added an `X` inside the checkbox.
Not sure what else has to be done.